app/services/doc_auth/error_generator.rb

```diff
@@ -339,8 +339,8 @@ def generate_doc_auth_errors(response_info)
       unknown_error_handler.handle(response_info).to_h
     end
 
-    def self.general_error(liveness_enabled)
-      liveness_enabled ? Errors::GENERAL_ERROR_LIVENESS : Errors::GENERAL_ERROR
+    def self.general_error(_liveness_enabled)
```

Comment:
Should we just remove this argument and update callsites?

```suggestion
    def self.general_error
```

Reply:
Or better yet: Get rid of this method and inline `Errors::GENERAL_ERROR` wherever it's called?